### DISCUSSION TITLE
Nested(multiple) crawling policy?

### DISCUSSION THREAD

**Discussion by flrngel** [DIYgod/RSSHub#13342] [Q&A]
Hi, I'm recently working on a [transform/sitemap](https://github.com/DIYgod/RSSHub/blob/master/lib/v2/rsshub/transform/sitemap.js)'s `sitemapindex` [protocol](https://www.sitemaps.org/protocol.html#sitemapIndex_sitemapindex) support. 

For example, [CNN's Sitemap Index](https://www.cnn.com/sitemaps/cnn/index.xml) is the case. This index contains multiple sitemaps. To build RSS of sitemaps' URLs, it needs to crawl the multiple URLs again.

I'm not sure if this is okay or not. Do you have any policy for this kind of pattern?

My current WIP code is [here](https://github.com/ainbr/RSSHub/commit/b6aa96ad4e726d522a4d7c96a1b51d17181753de).

**Comment by TonyRL** [comment]
It will be fine as long as the target site doesn't block you.

**Reply by flrngel** [reply]
Great. Will PR soon.
